Week3/homework/step2-2.js

```diff
@@ -3,18 +3,26 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let number = startIndex; number <= stopIndex; number++) {
+    numbers.push(number);
+  }
+  for (let i = 0; i < numbers.length; i++) {
+    const number = numbers[i];
```

Comment:
👍 for creating the 'convenience' variable so that you can use it instead of having to repeat `numbers[i]` throughout the loop body.